packages/common/src/enum/milestone-status.ts

```diff
@@ -65,3 +65,60 @@ export const COMPLETED_STATUSES = [
   STATUS.JOB_COMPETITION_CANCELLED,
   STATUS.JOB_OFFER_ACCEPTED,
 ];
+
+export enum StatusId {
+  COMPLETED_SLA = '9066b792-6fbf-4e60-803f-0554e4b4dba9',
```

Comment:
What I was thinking was to generate a map object from a database query.
The reason I was wary of your idea of changing status enum is that it is hard to notice whether there is an error or not.
Considering its complexity, uuid must never be meant to be written manually.
We may refactor this later.